src/BrazeKit-dev.js

```diff
@@ -287,6 +354,95 @@ var constructor = function () {
         }
     }
 
+    // mParticle commerce events use different appboy methods depending on if they are
+    // a purchase event or a non-purchase commerce event
+    function logCommerceEvent(event) {
+        var forwardProductsWithEvent =
+            forwarderSettings.forwardEnhancedECommerceData === 'True';
+
+        if (
+            event.EventCategory == mParticle.CommerceEventType.ProductPurchase
```

Comment:
Unless I'm missing something...

```suggestion
            event.EventCategory === mParticle.CommerceEventType.ProductPurchase
```